lib/network/modules/components/edges/util/EdgeBase.js

```diff
@@ -415,6 +415,9 @@ class EdgeBase {
       returnValue = Math.abs(Math.sqrt(dx * dx + dy * dy) - radius);
     }
 
+    // This is **wrong**! It doesn't take into account labels which are displayed rotated.
+    // It only works for horizontal labels.
+    // In addition, it's in violation of what this method is supposed to do.
```

Comment:
Should an issue be opened?

Reply:
Well spotted.

The consequence of removing this is that click will not work any more on a horizontal label in the regular click event, thus external changes. I'm a bit hesitant of removing it because users might expect it. I know, paranoia.

What's your opinion, given the comment? Can this be removed?

Reply:
I don't know, do we need to follow up on rotated labels?

Reply:
Clicking on rotated labels is the new bit, previously only clicking on horizontal labels worked.

Thinking about this, I realized that the question is: Should clicking on a node/edge label also select the given node/edge?

For labels within a node or on an edge, this is a no-brainer. For outside labels, less so.
I'm inclined to just remove that code, and rebuild the node selection around given method.

Also, I'll scan the issues to see what the users are expecting. Chances are someone has someone already has raised qualms about clicking labels to select.